### DISCUSSION TITLE
Supporting Vite

### DISCUSSION THREAD

**Discussion by RahulDey12** [spatie/laravel-multitenancy#375] [Q&A]
I was wondering how can we use muti tenancy with laravel's new vite compiler.

**Comment by dominikgeimer** [comment]
Did you run `npm run dev`? 

I am using Vite without any problem and I don't think that this is a package specific problem.

**Reply by RahulDey12** [reply]
It is fine for one URL defined in `APP_URL` but what about other urls it is not supporting HMR for other URLs
